### PR TITLE
[new] Add Kule Lazy

### PR DESCRIPTION
**Pull request by keicheng**

### CODE REVIEW

**Comment by keicheng** [comment]
i need to do something?

File: files/kule.lazy/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"packageManager": "github",
+	"name": "kule.lazy",
+	"repo": "keicheng/kule.lazy",
+	"files": {
+		"include": ["kule-lazy-full.min.css", "kule-effects.min.css", "kule-animates.min.css", "kule-reset.min.css", "kule-grid.min.css"]
```
Comment (PeterDaveHello) [inline files/kule.lazy/update.json]: These files are not directly located under the root of the project's repository, may need to set `basePath` for it.

Ref:
https://github.com/jsdelivr/libgrabber#add-updatejson-schema

Comment (keicheng) [inline files/kule.lazy/update.json]: ok, thanks

File: files/kule.lazy/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+	"packageManager": "github",
+	"name": "kule.lazy",
+	"repo": "keicheng/kule.lazy",
+	"files": {
+		"basePath": "/",
+		"exclude": ["css/**/*"]
```
Comment (jimaek) [inline files/kule.lazy/update.json]: This makes no sense. Please read our documentation and fix the rules

**Pull request closed without merging**